src/parsers/ddi_parser.jl

```diff
@@ -0,0 +1,174 @@
+
+
+"""
+    parse_ddi(filepath::String)
+
+Parses a valid IPUMs DDI XML file and returns a DDIInfo object containing 
```

Comment:
Tiny comment: could you put DDInfo in graveticks (i.e. `DDIInfo`)? I think this allows Documenter to cross-link within documentation.

Reply:
This should now be fixed.